test/functional/mempool_spend_coinbase.py

```diff
@@ -28,14 +28,14 @@ def run_test(self):
         chain_height = 198
         self.nodes[0].invalidateblock(self.nodes[0].getblockhash(chain_height + 1))
         assert_equal(chain_height, self.nodes[0].getblockcount())
+        wallet.rescan_utxos()
 
         # Coinbase at height chain_height-100+1 ok in mempool, should
         # get mined. Coinbase at height chain_height-100+2 is
         # too immature to spend.
-        wallet.scan_blocks(start=chain_height - 100 + 1, num=1)
-        utxo_mature = wallet.get_utxo()
-        wallet.scan_blocks(start=chain_height - 100 + 2, num=1)
-        utxo_immature = wallet.get_utxo()
+        coinbase_txid = lambda h: self.nodes[0].getblock(self.nodes[0].getblockhash(chain_height - 100 + h))['tx'][0]
+        utxo_mature = wallet.get_utxo(txid=coinbase_txid(1))
+        utxo_immature = wallet.get_utxo(txid=coinbase_txid(2))
```

Comment:
nit, feel free to ignore: For increased readability, I'd rather prefer explicitely passing the absolute heights here, instead of putting magic into the lambda.

Reply:
Thanks, done.